_docs/setup/kubernetes/ansible-install.md

```diff
@@ -0,0 +1,121 @@
+---
+title: Installing with Ansible
+overview: Instructions on using the included Ansible playbook to perform installation.
+
+order: 40
+
+layout: docs
+type: markdown
+---
+{% include home.html %}
+
+The Ansible scenario defined within this project will allow you to : 
+
+- Install `Istio` Distribution and set the path of the `istioctl` go client (if you execute the command manually)
```

Comment:
I don't quite understand the "if you execute the command manually.  Could you expand on the statement or alternatively delete it?